### PR TITLE
cake-5294 | Set up ddb affiliate units

### PR DESCRIPTION
**Pull request by cschucode**
## Links

* http://wikia-inc.atlassian.net/browse/CAKE-5294

## Description

Create DDB affiliate units in JSON file and add targeting info.

Note: 
Due to the increase in size of JSON files, bundle size was increase by 1KB.

## Reviewers

@Wikia/cake

### CODE REVIEW

File: app/services/affiliate-slots-units.json [hunk 1]
```diff
@@ -211,5 +211,28 @@
...
+  {
+    "campaign": "ddb",
+    "category": "ddb",
+    "tagline": "Related Posts",
+    "image": "https://static.wikia.nocookie.net/0cebb500-f8c3-4f44-b4aa-f44e33fc2319",
+    "heading": "The Official Digital Toolset for Dungeons & Dragons",
+    "subheading": "Visit D&D Beyond",
+    "launchOn": "2019-12-12T00:00:00Z",
+    "link": "https://www.dndbeyond.com/intro",
+    "links": null
```
Comment (mklucsarits) [inline app/services/affiliate-slots-units.json]: Can we just leave this out? Or it has to be `null`?

**Review by mklucsarits — APPROVED** [review]

File: app/services/affiliate-slots-units.json (hunk 1 above)
Comment (cschucode) [reply app/services/affiliate-slots-units.json]: Good call. 4f3dc48

File: app/services/affiliate-slots-targeting.json [hunk 2]
```diff
@@ -41,5 +41,12 @@
     "wikiId": [
       673
     ]
+  },
+  {
+    "campaign": "ddb",
+    "category": "ddb",
+    "wikiId": [
+      51
```
Comment (vforge) [inline app/services/affiliate-slots-targeting.json]: What wiki is this?

File: app/services/affiliate-slots-units.json [hunk 3]
```diff
@@ -211,5 +211,26 @@
...
     "isBig": true
+  },
+  {
+    "campaign": "ddb",
+    "category": "ddb",
+    "tagline": "Related Posts",
+    "image": "https://static.wikia.nocookie.net/0cebb500-f8c3-4f44-b4aa-f44e33fc2319",
+    "heading": "The Official Digital Toolset for Dungeons & Dragons",
+    "subheading": "Visit D&D Beyond",
+    "launchOn": "2019-12-12T00:00:00Z",
```
Comment (vforge) [inline app/services/affiliate-slots-units.json]: Is that correct? It's 12/11 4pm PDT.

Comment (cschucode) [reply app/services/affiliate-slots-units.json]: No. Definitely 12/12. Will get a specific time.

Comment (cschucode) [reply app/services/affiliate-slots-units.json]: From Keiko, going with 11am (PST) on Dec. 12, 2019.

File: app/services/affiliate-slots-targeting.json (hunk 2 above)
Comment (cschucode) [reply app/services/affiliate-slots-targeting.json]: Just realized this was for filtering. Will remove because will be driven by Taxonomy ™️

**Review by vforge — APPROVED** [review]

**Pull request merged**